apps/omg_watcher/test/integration/invalid_exit_test.exs

```diff
@@ -65,19 +65,20 @@ defmodule OMG.Watcher.Integration.InvalidExitTest do
         alice.addr
       )
 
-    {:ok, %{"status" => "0x1"}} = Eth.WaitFor.eth_receipt(txhash, @timeout)
-
-    Process.sleep(1_000)
+    # TODO: make event payload testing approximate not exact, so that we needn't parse
+    {:ok, %{"status" => "0x1", "blockNumber" => "0x" <> eth_height}} = Eth.WaitFor.eth_receipt(txhash, @timeout)
+    {eth_height, ""} = Integer.parse(eth_height, 16)
 
     invalid_exit_event =
       Client.encode(%Event.InvalidExit{
         amount: 10,
         currency: @eth,
         owner: alice.addr,
-        utxo_pos: utxo_pos
+        utxo_pos: utxo_pos,
+        eth_height: eth_height
       })
 
-    assert_push("invalid_exit", ^invalid_exit_event)
+    assert_push("invalid_exit", ^invalid_exit_event, 5_000)
```

Comment:
why only here it required to change the timeout to 5_000 for `assert_push`?

Reply:
only here we don't know how long to wait exactly, so the custom timeout is useful to not wait anything extra. In other cases, we're waiting on other things (a particular child block validation, `BlockGetter` stopping) so the timeout is not necessary.